tests/web/iqr_service/test_iqr_service.py

```diff
@@ -58,27 +59,18 @@ def setUp(self) -> None:
         self.app = IqrService(config)
 
     def assertStatusCode(self, r: TestResponse, code: int) -> None:
-        """
-        :type r: :type: flask.wrappers.Response
-        :type code: int
-        """
         self.assertEqual(code, r.status_code)
 
     def assertJsonMessageRegex(self, r: TestResponse, regex: str) -> None:
-        """
-        :type r: flask.wrappers.Response
-        :type regex: str
-        """
         self.assertRegex(json.loads(r.data.decode())['message'], regex)
 
     # Test Methods ############################################################
 
     def test_is_ready(self) -> None:
         # Test that the is_ready endpoint returns the expected values.
-        #: :type: flask.wrappers.Response
-        r = self.app.test_client().get('/is_ready')
-        self.assertStatusCode(r, 200)
-        self.assertJsonMessageRegex(r, "Yes, I'm alive.")
+        r: flask.wrappers.Response = self.app.test_client().get('/is_ready')  # type: ignore
+        self.assertStatusCode(r, 200)  # type: ignore
```

Comment:
Its more probably a mistake that the `assertStatusCode` should be annotated with `Response` instead of `TestResponse`.

Reply:
Thanks! I changed import and annotations to `Response`. mypy is passing now.